src/escpos/escpos.py

```diff
@@ -1494,6 +1507,204 @@ def buzzer(self, times: int = 2, duration: int = 4) -> None:
 
         self._raw(BUZZER + six.int2byte(times) + six.int2byte(duration))
 
+    def _enter_kanji_mode(self):
+        """Enter Kanji mode."""
+        self._raw(KANJI_ENTER_KANJI_MODE)
+
+    def _exit_kanji_mode(self):
+        """Exit Kanji mode."""
```

Comment:
```suggestion
    def _exit_kanji_mode(self) -> None:
```
Please annotate a return type None for completeness.